tests/test_configuration.py

```diff
@@ -18,15 +18,14 @@
 # Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 #
 import unittest
-from unittest.mock import patch, Mock
+from unittest.mock import patch
 from py_sonar_scanner.configuration import Configuration
 
 
 class TestConfiguration(unittest.TestCase):
     @patch("py_sonar_scanner.configuration.sys")
     def test_argument_parsing_empty_toml(self, mock_sys):
```

Comment:
So I had this test function made such that we don't test reading from toml files here. Since now you added test cases that do so, I think the function name is a bit misleading. Would it be possible to rename this function, or move the tests to a different function?